acmw-db-app/pages/api/mailer.js

```diff
@@ -0,0 +1,16 @@
+// using Twilio SendGrid's v3 Node.js Library
+// https://github.com/sendgrid/sendgrid-nodejs
+const sgMail = require('@sendgrid/mail');
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+const sendEmail = (email, token) => {
+    const msg = {
+    to: email,
+    from: 'acmwosu@gmail.com',
+    subject: 'ACM-W OSU Database - Reset Password',
+    text: 'hellooooo, here\'s your token: ' + token
+  };
+  sgMail.send(msg);
+}
+const email = '';
+sendEmail(email, 'cat')
```

Comment:
Since this is an HTTP request endpoint there should be a response given

Reply:
I feel like. I should delete that part, so we can just export the `sendEmail` function and use it in `reset.js`?

Reply:
yeah I think that's fine if you just want to delete that file and move the code to the reset file